docs/docs-ref-autogen/outlook/office.appointmentread.yml

```diff
@@ -555,6 +555,26 @@ items:
       level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)</td> <td>ReadItem</td>
       </tr> <tr> <td>[Applicable Outlook mode](https://docs.microsoft.com/outlook/add-ins/#extension-points)</td>
       <td>Appointment Attendee</td> </tr> </table>
+
+      #### Examples
+
+
+      ```javascript
+
+      Office.context.mailbox.item.enhancedLocation.getAsync(callbackFunction);
+
+
+      function callbackFunction(asyncResult) {
+          asyncResult.value.forEach(function (place) {
+                  console.log("Display name: " + place.displayName);
+                  console.log("Type: " + place.locationIdentifier.type);
+                  if (place.locationIdentifier.type == Office.MailboxEnums.LocationType.Room) {
+                      console.log("Email address: " + place.emailAddress);
+              }
+          });
+      }
```

Comment:
Need to fix the indentation in this code sample (in the snippet file where this code sample originates from) -- the first several lines within the forEach are indented 4 spaces too far.